sensor_array/helper/generate_comps_for_simulations.py

```diff
@@ -19,12 +19,14 @@ def create_henrys_comp_list(henrys_gases, henrys_comps, background_gases, backgr
         comp_list_temp.extend(bg_comp)
         comp_list.append(comp_list_temp)
 
-    if filename != None:
-        gases = henrys_gases+background_gases
-        with open(filename, 'w', newline='') as csvfile:
-            writer = csv.writer(csvfile, delimiter='\t', quotechar='|', quoting=csv.QUOTE_MINIMAL)
-            writer.writerow(gases)
-            writer.writerows(comp_list)
+    if filepath != None:
+        for gas in henrys_gases:
+            gases = [gas]+background_gases
+            filename = filepath+gas+'.csv'
+            with open(filename, 'w', newline='') as csvfile:
+                writer = csv.writer(csvfile, delimiter='\t', quotechar='|', quoting=csv.QUOTE_MINIMAL)
+                writer.writerow(gases)
+                writer.writerows(comp_list)
```

Comment:
How much did this speed things up?

Reply:
It's not noticeable, but its technically more efficient since the composition set only gets created once.